ConwaysGameOfLife/GameController.cs

```diff
@@ -70,43 +67,40 @@ public void Run()
             _output.WriteLine(Messages.Welcome);
             // create initial universe
             CreateInitialUniverse();
+            DisplayUniverse();
             // add live cells to universe until user presses 'p' to play
+            // PopulateUniverseWithLiveCells();
+            // DisplayUniverse();
             // generator checks all cells if change of state required & creates next universe
             // loop last step until user presses 'q' to quit or all cells are dead
         }
-        
+
         public string CreateValidUniverseString()

```

Comment:
this method can be private. When you test the CreateInitialUniverse method, it will also be testing this method as it's used in that method. In saying that, see Keddy's comments about testing the run method. If you decide to make this method private too, you can still test all the functionality by testing the run method.